hr_attendance_calendar/wizards/hr_attendance_settings.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Stephane Eicher <seicher@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, fields, api
+
+
+class HrAttendanceSettings(models.TransientModel):
+    """ Settings configuration for hr.attendance."""
+    # _name = "hr.attendance.settings"
+    _inherit = 'base.config.settings'
+
+    work_day_duration = fields.Float('Work day duration (hour)')
```

Comment:
This means that every employee has to work the same time ? Shouldn't we get the duration from the contract of the employee ?